src/main/java/com/ita/if103java/ims/controller/UserController.java

```diff
@@ -91,12 +79,13 @@ public boolean delete(@PathVariable("id") Long id) {
     @ApiPageable
```

Comment:
Rostyk is going to remove this annotation. Please review his PR
https://github.com/IF-103-Java/ims/pull/170

And @ApiIgnore below will not be needed too.